src/app/access-requests/features/dynamic-access-request-button/dynamic-access-request-button.component.html

```diff
@@ -0,0 +1,17 @@
+@switch (status()) {
+  @case ('pending') {
+    <button mat-raised-button [disabled]="true">
+      <mat-icon>key</mat-icon> Pending Access Request ...
+    </button>
+  }
+  @case ('allowed') {
+    <button mat-raised-button routerLink="/work-package">
+      <mat-icon>key</mat-icon>Download the Data
```

Comment:
```suggestion
      <mat-icon>key</mat-icon>Download Data
```
Using 'the Data' sounds wrong, but Download Data is also not perfect.

Reply:
"Download Date" gives me 10x more google search results than "Download Data" so I would leave it at this for now.